xtext-website/_posts/releasenotes/2020-06-02-version-2-22-0.md

```diff
@@ -16,6 +16,8 @@ As you might have recognized, the number of people contributing to Xtext on a re
 
 The [Xtext community website](https://www.eclipse.org/Xtext/community.html) and the [Xtext documentation website](https://www.eclipse.org/Xtext/documentation/index.html) have been updated. New Xtext-based projects have been added, some editor features and the Unit testing / UI testing capabilities have been documented and the embedded code snippets have been updated as well.
 
+Documentation about [regenerating the test- and production Xtext languages](https://github.com/eclipse/xtext/blob/master/dev-doc/RegenerateLanguages.md) was added to enable more people to contribute to Xtext.
```

Comment:
am not sure if this is relevant to end users at all

Reply:
I think it is a good opportunity to make the Xtext users aware that such a `dev-doc `documentation exists and where it is located.

Reply:
maybe you can generalize this more (e.g. enhanced developer documentation at ....

Reply:
Good idea! I updated the PR accordingly.